resources/user-agents/browsers/palemoon/palemoon-20-25.json

```diff
@@ -1,6 +1,6 @@
 {
   "division": "PaleMoon #MAJORVER#.#MINORVER#",
-  "versions": ["25.9", "25.8", "25.7", "25.6", "25.5", "25.4", "25.3", "25.2", "25.1", 25, "24.9", "24.8", "24.7", "24.6", "24.5", "24.4", "24.3", "24.2", "24.1", 24, "20.3", "20.2", "20.1", 20],
```

Comment:
I did remove some versions to make this better align with what is listed in PaleMoon's FTP archive (ftp://archive.palemoon.org/) the only one that I left that isn't in their FTP is 24.9, which browscap has a test for.  I did search a bit to see if it was actually a release that didn't make it to FTP for some reason, but nothing concrete came up. There was some discussion about PaleMoon 25.1 acting as Firefox 24.9, but that seemed related to addons.mozilla.org and not really the useragent.  I left the test and the version anyway.